alert-policies/flutter-web/ApplicationErrorRate.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: Application Error Rate
+
+# Description and details
+description: |+
+  This alert is triggered when the application error rate are higher than 5 for 5 minutes.
```

Comment:
This alert is triggered when the application error rate is higher than 5 for 5 minutes.

Reply:
Hi @pkosanam,
Updated alert description. kindly review it.